Exploring_Pegasus/get_inactive_qubits.py

```diff
@@ -0,0 +1,27 @@
+# Copyright 2020 D-Wave Systems Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import dwave_networkx as dnx
+from dwave.system.samplers import DWaveSampler
+
+dwave_sampler_pegasus = DWaveSampler(solver={'topology__type__eq': 'pegasus'})
+props_pegasus = dwave_sampler_pegasus.properties
+
+# Get total qubits - should be 24 * N * (N - 1)
+total_qubits = props_pegasus['num_qubits']
```

Comment:
An alternative (via lower-level cloud client `Solver` object):
```
num_qubits = dwave_sampler_pegasus.solver.num_qubits
```
For active only:
```
num_active_qubits = dwave_sampler_pegasus.solver.num_active_qubits
```

Reply:
I did change the first one. But on the second, 60 of the 5510 active qubits are turned off, because they're disconnected. I"m focusing on 'available' rather than 'active'. Let me know what you think.

Reply:
Not sure we use it consistently across Ocean (@JoelPasvolsky?), but these are [properties](https://docs.ocean.dwavesys.com/en/stable/docs_cloud/reference/solver.html#properties) in the cloud client.

Reply:
We have not, and should start

Reply:
FWIW, this is the terminology we use in the cloud client:

![image](https://user-images.githubusercontent.com/447371/94155812-363ddd80-fe34-11ea-91d1-6c43878fb893.png)

Reply:
I added your approach, too. I was getting different answers from num_active_qubits a couple of days ago, but now they're in synch. (In other words, now the functional but turned off qubits are NOT counted here. The user can't use them. This is now correct).